test/Benchmarks/OrleansBenchmarks/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "dependencies": {
+    "BenchmarkDotNet": "0.9.0",
+    "System.Collections": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Threading": "4.0.0",
+    "System.Threading.Tasks": "4.0.0"
```

Comment:
I believe that when using net451 as the framework, you shouldn't need to reference these that are automatically included in the framework (and conflict with the versions from the framework).
I'm not entirely familiar with this yet, but take a look at the log: http://dotnet-ci.cloudapp.net/job/dotnet_orleans/job/innerloop_prtest/1785/consoleFull

And search for "Detected package downgrade". You'll see a ton of these:
```
14:29:55      1>EXEC : warning : Detected package downgrade: System.Collections from 4.0.11 to 4.0.0 [D:\j\workspace\innerloop_prtest933994bb\src\Orleans.sln]
14:29:55           OrleansBenchmarks (>= 1.0.0) -> Tester (>= 1.0.0) -> Microsoft.Extensions.DependencyInjection (>= 1.0.0) -> System.Collections (>= 4.0.11) 
14:29:55           OrleansBenchmarks (>= 1.0.0) -> System.Collections (>= 4.0.0)
14:29:55      1>EXEC : warning : Detected package downgrade: System.Threading from 4.0.11 to 4.0.0 [D:\j\workspace\innerloop_prtest933994bb\src\Orleans.sln]
14:29:55           OrleansBenchmarks (>= 1.0.0) -> Tester (>= 1.0.0) -> Microsoft.Extensions.DependencyInjection (>= 1.0.0) -> System.Threading (>= 4.0.11) 
14:29:55           OrleansBenchmarks (>= 1.0.0) -> System.Threading (>= 4.0.0)
<snip for brevity/>
```